manifest-dev.yml

```
@@ -10,7 +10,7 @@ applications:
     ACCOUNT_STORE_API_HOST: https://funding-service-design-account-store-dev.london.cloudapps.digital
     APPLICATION_STORE_API_HOST: https://funding-service-design-application-store-dev.london.cloudapps.digital
     NOTIFICATION_SERVICE_HOST: http://funding-service-design-notification-dev.apps.internal:8080
-    MAGIC_LINK_REDIRECT_URL: https://funding-service-design-frontend-dev.london.cloudapps.digital/account/{account_id}
+    MAGIC_LINK_REDIRECT_URL: https://frontend.dev.fundingservice.co.uk/account
```

```suggestion
    APPLICANT_FRONTEND_HOST: https://frontend.dev.fundingservice.co.uk
```
I think this is a bit simpler and means we can rely on the config for the path, this is what we do in the docker runner: https://github.com/communitiesuk/funding-service-design-docker-runner/blob/00381119ef3b76d75812129435c28f9396c5ae98/docker-compose.yml#L43

---

Agreed - have simplified the code around this too